src/components/oc-client/src/oc-client.js

```diff
@@ -294,25 +310,35 @@ var oc = oc || {};
 
   oc.render = function(compiledViewInfo, model, callback){
     oc.ready(function(){
-      if(!!compiledViewInfo.type.match(/jade|handlebars/g)){
+
+      var template = templates.find(function(template){return template.type === compiledViewInfo.type});
```

Comment:
find is not supported in IE: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find
Perhaps we can use something available in jQuery?